kronofoto/archive/templates/archive/photo_detail.html

```diff
@@ -47,14 +47,14 @@
             <script type="text/javascript">
               var addZoom = (target) => {
 
-                let container = document.getElementById(target),
+                let container = window.kfcontext.getElementById(target),
```

Comment:
`window.kfcontext` is still not a solution. This will not work if there is more than one fortepan-viewer on the page, or if there is ever a preview fortepan-viewer within fortepan.us. The solution is to put some `data-zoom-target` or something on an attribute, find and process it in htmx.onLoad. That is the case everywhere `window` is being used to store a global reference.